internal/cmd/resourcemanager.go

```diff
@@ -110,7 +117,7 @@ func runResourceTop(options *resourceTopOptions) func(cmd *cobra.Command, args [
 	}
 }
 
-func aggregateByUser(providers []coder.KubernetesProvider, users []coder.User, orgs []coder.Organization, envs []coder.Environment, options resourceTopOptions) ([]groupable, envLabeler) {
+func aggregateByUser(providers []coder.KubernetesProvider, users []coder.User, orgs []coder.Organization, envs []coder.Environment, images map[string]*coder.Image, options resourceTopOptions) ([]groupable, envLabeler) {
```

Comment:
Nit: The amount of params we're passing here seems to be becoming unwieldy.

I think a struct for collecting these could provide useful, especially if we can reuse it for some of these other `aggregateX` funcs

Reply:
true true, adding now